src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -383,6 +383,6 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
     ChangeInfo {
         change_id: 137147,
         severity: ChangeSeverity::Info,
-        summary: "New option `build.exclude` that adds support for excluding test.",
+        summary: "Added new option `build.exclude` which works the same way as `--exclude` flag on `x`.",
```

Comment:
There's a "the" missing here but I guess it's not worth another PR.^^
(should be "same as as *the* `--exclude` flag")